doc.go

```diff
@@ -161,6 +161,40 @@ setting a timeout on the context passed to NewClient. Dialing is non-blocking,
 so timeouts would be ineffective and would only interfere with credential
 refreshing, which uses the same context.
 
+# Headers
+
+Regardless of which transport is used, request headers can be set in the same
+way using [`callctx.SetHeaders`][setheaders].
+
+Here is a trivial example:
+
+	// Set the header "key" to "value".
+	ctx := callctx.SetHeaders(context.Background(), "key", "value")
+
+	// Then use ctx in a subsequent request.
+	response, err := client.GetSecret(ctx, request)
+
+## Google-reserved Headers
+
+There are a some header keys that Google reserves for internal use that must
+not be ovewritten. The following header keys are broadly considered reserved
+and should not be conveyed by client library users unless instructed to do so:
+
+* `x-goog-api-client`
+* `x-goog-request-params`
+
+Be sure to check the individual package documentation for other service-specific
+reserved headers.
+
+## Google Cloud System Parameters
+
+Google Cloud services respect a well defined set of so called
+[system parameters][system parameters]. They can be used to augment request
+and/or response behavior. For the most part, they are not needed when using one
```

Comment:
Google Cloud services respect [system parameters][system parameters] that can be used to augment request
and/or response behavior.

Reply:
Done plus some line (re)wrapping